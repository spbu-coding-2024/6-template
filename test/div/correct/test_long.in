806954824942951088906984201526885759422639452418485387510625454092303185918649290564931015996917903564963469881635875115018000688681500336883650312858964396320237832003596585522533565843321175678870816652376469687367407647865509307239936654499206504680797288163273990211524854981698270639538018054427041432874577963002059072735950587832500247141644354279261774732771194812989879219685197695101270323705570323360939184113918438525993732922113398359953767825451478406733031069168640281773377237679922429396606918241997181515356337055270943459940045 -8187254825947690889831350828548747397478086365586218884419012879001179933503367752948348743537867058508960895334498593154471944282781230922981327704438179775828848888105273335694494127581338691810329287059218250628003084872047412424006147309417166333982888677223817209723203637261583287880205436410158311792260858514747136022569084641026233756657741105696482335855796671595167066427477908048992352642528862994136196952859965163456052211918822198084531608238130634322535874930104702376237017735005266763500840219561670 /
