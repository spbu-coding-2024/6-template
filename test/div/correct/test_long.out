-98562319372970594787438036059
